bob-common/src/data.rs

```diff
@@ -92,26 +94,44 @@ pub type VDiskId = u32;
 
 #[derive(Clone)]
 pub struct BobData {
-    inner: Vec<u8>,
+    inner: Bytes,
     meta: BobMeta,
 }
 
 impl BobData {
-    pub fn new(inner: Vec<u8>, meta: BobMeta) -> Self {
+    const TIMESTAMP_LEN: usize = 8;
+    
+    pub fn new(inner: Bytes, meta: BobMeta) -> Self {
         BobData { inner, meta }
     }
 
     pub fn inner(&self) -> &[u8] {
         &self.inner
     }
 
-    pub fn into_inner(self) -> Vec<u8> {
+    pub fn into_inner(self) -> Bytes {
         self.inner
     }
 
     pub fn meta(&self) -> &BobMeta {
         &self.meta
     }
+
+    pub fn to_vec(&self) -> Vec<u8> {
```

Comment:
I suggest to update the name to indicate that this function does not just return a buffer of data, but also includes a timestamp in the data. Possible name: `to_serialized_vec`